### ISSUE TITLE
How to download the feature database

### ISSUE THREAD

**Issue by Mareeta26** [hmmlillian/Gray-Image-Retrieval#4]
Hi,
I can't download the compressed feature database from https://drive.google.com/uc?id=1FS04QjIowFN2iFWqP_5lfcG6GyGH5yJE&export=download

Is it possible to give another link or upload the database?

May thanks!

**Comment by CybotDNA** [comment]
yes, the link is down

**Comment by hmmlillian** [comment]
> yes, the link is down

I tried. It works. Did you meet any problem?

**Comment by CybotDNA** [comment]
> 
> 
> > yes, the link is down
> 
> I tried. It works. Did you meet any problem?

ok, it doesn't seem to be down, but it says it can't be displayed because the file is too big, or too many people are accessing it.

**Comment by hmmlillian** [comment]
> > > yes, the link is down
> > 
> > 
> > I tried. It works. Did you meet any problem?
> 
> ok, it doesn't seem to be down, but it says it can't be displayed because the file is too big, or too many people are accessing it.

Yes, it shows "Google Drive can't scan this file for viruses." but there is a button "Download anyway" below. If you click it, then a window "Save as" will pop out after a while. The link does work.
